src/universal-dapp.js

```diff
@@ -290,12 +290,18 @@ UniversalDApp.prototype.getCallButton = function(args) {
         return $('<div class="' + returnCls + '">').html('<strong>' + returnName + ':</strong> ' + JSON.stringify( result, null, 2 ) )
     }
 
-    var getGasUsedOutput = function (result) {
+    var getGasUsedOutput = function (result, vmResult) {
         var $gasUsed = $('<div class="gasUsed">')
         var caveat = lookupOnly ? '<em>(<span class="caveat" title="Cost only applies when called by a contract">caveat</span>)</em>' : '';
         if (result.gasUsed) {
             var gas = result.gasUsed.toString(10)
-            $gasUsed.html('<strong>Cost:</strong> ' + gas + ' gas. ' + caveat )
+            $gasUsed.html('<strong>Transaction cost:</strong> ' + gas + ' gas. ' + caveat )
+        }
+        if (vmResult.gasUsed) {
+            var $callGasUsed = $('<div class="gasUsed">')
+            var gas = vmResult.gasUsed.toString(10)
+            $callGasUsed.append('<strong>Inter-contract call cost:</strong> ' + gas + ' gas.' )
```

Comment:
But then even more so an argument for something like `gas used: x + y (execution + transaction)` in the vm mode and `gas used: x` in the web3 mode.